examples/logger/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog for `logger-example`
```

Comment:
Do we need this file?

Reply:
No, we don't need it. Let me remove this file.